EXT_BeamPhysics.md

```diff
@@ -0,0 +1,252 @@
+Extension to the openPMD Standard for Describing Particle Beams and X-rays
+==========================================================================
+
+Version 2.0.0
+
+Overview
+--------
+
+The `BeamPhysics` extension to the openPMD standard is meant for describing particles and fields commonly encountered
```

Comment:
feel free to keep latex-like one sentence per line (preferred).
(Alternatively, please break all at 80 char.)

Reply:
@ax3l 
Right now mostly long paragraphs are a single line. I don't mind to break at 80 characters but I do not know how to do convert the file efficiently.

Reply:
I mean: just keep one sentence per line and break after the sentence.
This will keep diffs smaller and is the least stressful while writing and editing.

Reply:
OK

Reply:
somehow this is still two lines :)

Reply:
Done.